config/validate/validate.go

```diff
@@ -77,13 +78,12 @@ func validateCloudConfig(config []byte, rules []rule) (report Report, err error)
 // any parsing issues into the provided report. Unrecoverable errors are
 // returned as an error.
 func parseCloudConfig(cfg []byte, report *Report) (node, error) {
-	yaml.UnmarshalMappingKeyTransform = func(nameIn string) (nameOut string) {
-		return nameIn
-	}
+	cfg = []byte(config.Normalize(bytes.NewReader(cfg)))
```

Comment:
Hm, this normalizes the yaml config now, whereas the old code had a no-op transform at this point. Not sure if it matters, though.